lib/rspec/core/project_initializer/spec/spec_helper.rb

```diff
@@ -66,12 +66,6 @@
   # particularly slow.
   config.profile_examples = 10
 
-  # Run specs in random order to surface order dependencies. If you find an
-  # order dependency and want to debug it, you can fix the order by providing
-  # the seed, which is printed after each run.
-  #     --seed 1234
-  config.order = :random
-
```

Comment:
I would instead explain that random is the default, but you can switch back to defined, or another ordering scheme.

Reply:
I agree, I didn't do it because at the top of this commented configs it is saying that these are "suggested settings", which is no longer true if we have an example talking about setting `config.order = :defined`, right?

```
# The settings below are suggested to provide a good initial experience
# with RSpec, but feel free to customize to your heart's content.
```